source/context.ts

```diff
@@ -356,7 +356,12 @@ export class Context {
             return;
         }
 
+        // Only handle masquerade here and not within each supports-query?
```

Comment:
@scheibel remove this comment, or just the question mark?

Reply:
This is still undecided. As of now, the masquerade is also checked within each `supports` query (https://github.com/cginternals/webgl-operate/blob/22ddc5cf309cc0ffa7b021b4f2465782e75f935e/source/context.ts#L299).